target/linux/ath79/image/generic.mk

```diff
@@ -1242,6 +1287,15 @@ define Device/dlink_dir-859-ax
   IMAGE_SIZE := 15872k
   DEVICE_PACKAGES := kmod-usb2 kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct
   SEAMA_SIGNATURE := wrgac37_dlink.2013gui_dir859
+  IMAGES += factory.bin factory2.bin recovery.bin
+  IMAGE/recovery.bin := $$(IMAGE/default) | pad-rootfs -x 64 | seama | \
+	seama-seal | check-size
+  IMAGE/factory.bin := $$(IMAGE/recovery.bin) | \
```

Comment:
Even the latest firmware of dir-859 is still the legacy seama header. Are you sure we need this?
https://media.dlink.eu/support/products/dir/dir-859/driver_software/dir-859_fw_reva_1-05b03_eu_multi_20161102.zip

Reply:
I'm not sure what is the bottom line. What should be changed there?

Reply:
Nothing to change here, I just confirmed the encrypted image is indeed required for this device, since v1.07 is (among) the latest OEM firmwares available, and only accepts encrypted images (while 1.05 was still using the unencrypted (recovery) format).